packages/backend/package.json

```diff
@@ -7,15 +7,14 @@
     "bugs": "https://github.com/quadratic-funding/mpc-phase2-suite/issues",
     "license": "MIT",
     "private": false,
-    "main": "dist/src/functions/index.node.js",
+    "main": "./dist/src/functions/index.node.js",
     "exports": {
         "import": "./dist/src/functions/index.node.mjs",
         "require": "./dist/src/functions/index.node.js"
     },
-    "type": "module",
     "types": "dist/types/index.d.ts",
     "engines": {
-        "node": ">=16.14.0"
+        "node": "16"
```

Comment:
It will fails to build on Node.js version greater than 16 (e.g. v18). Are we going to support only v16?

Reply:
Ok. Opened PRs reflects it:
* https://github.com/quadratic-funding/mpc-phase2-suite/pull/228
* https://github.com/quadratic-funding/mpc-phase2-suite/pulls

Reply:
Thanks for taking care of this, will review the PRs when ready.